sentry_sdk/integrations/opentelemetry/span_processor.py

```diff
@@ -116,25 +114,23 @@ def _prune_old_spans(self):
 
     def on_start(self, otel_span, parent_context=None):
         # type: (OTelSpan, Optional[SpanContext]) -> None
-        hub = Hub.current
-        if not hub:
```

Comment:
Does the `dsn` check make this check redundant? Trying to understand why we are removing this

Reply:
What would be the post-hub world equivalent of this check? It seems to me like the purpose of the hub in this function was basically two things: accessing the client and making sure the client has a DSN, and using the hub directly when starting the Sentry span (which we don't want to do anymore in favor of using the top-level API).

Reply:
As far as I remember those checks where here to just make sure we have a Sentry setup with and DSN where we can send data to.

So I think it is OK to remove the hub check and just check for a valid DSN further down.